jbpm-db-scripts/src/main/resources/db/upgrade-scripts/db2/jbpm-7.60-to-7.61-TimerMappingInfo.sql

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Just curious, why are we naming this file with suffix `TimerMappingInfo`? is it because this is optional?